src/pykeen/datasets/openea.py

```diff
@@ -133,8 +133,8 @@ def _load(self) -> None:
 
         # ensure file is present
         if not path.is_file() or self.force:
-            logger.info(f"Downloading file from Dropbox (Link: {self.__class__.DROPBOX_LINK})")
-            download(url=self.__class__.DROPBOX_LINK, path=path, hexdigests={"sha512": self.SHA512})
+            logger.info(f"Downloading file from Figshare (Link: {self.__class__.FIGSHARE_LINK})")
+            download(url=self.__class__.FIGSHARE_LINK, path=path, hexdigests={"sha512": self.SHA512})
```

Comment:
Can we combine `download` + `read_zipfile_csv` by using [`pystow.ensure_zip_df`](https://pystow.readthedocs.io/en/latest/api/pystow.ensure_zip_df.html#pystow.ensure_zip_df)?

Reply:
I originally wanted to do that, but I was unsure how to combine this with the `_help_cache` method. And if I start to mangle around with the `self.cache_root` path it would undermine using `self._help_cache` in the first place.

Reply:
@cthoyt , it may be better to use `pystow`s submodules instead of resolving to a path too early. This is something for a separate PR though.